### PR TITLE
Remove covers-validator in CI

### PR DESCRIPTION
**Pull request by gbirke**
This functitionality is integrated into PHPUnit since version 5.2.6

Require PHPUnit explicitly.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	"require-dev": {
 		"squizlabs/php_codesniffer": "~2.2",
 		"phpmd/phpmd": "~2.3",
-		"ockcyp/covers-validator": "~0.4.0"
+		"phpunit/phpunit": "~5.4"
```
Comment (sebastianbergmann) [inline composer.json]: To make the Travis build work again for PHP 5.5 this needs to be changed to `"~4.8 || ~5.4"`.

Invalid `@covers` annotations will be break the build with PHPUnit 4.8, though.

Does this component have to be compatible with PHP 5, especially an unsupported version of PHP 5, though?

Comment (manicki) [inline composer.json]: It should be compatible with PHP 5.6, I think. PHP 5.5 requirement is obsolete.
See: https://github.com/wmde/FundraisingStore/pull/87

File: composer.json [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
+		"php": "~5.5||~7.0",
 		"doctrine/dbal": "~2.4",
 		"doctrine/orm": "~2.4",
 		"gedmo/doctrine-extensions": "^2.4"
 	},
 	"require-dev": {
 		"squizlabs/php_codesniffer": "~2.2",
 		"phpmd/phpmd": "~2.3",
-		"ockcyp/covers-validator": "~0.4.0"
+		"phpunit/phpunit": "~4.8||~5.4"
```
Comment (JeroenDeDauw) [inline composer.json]: > since version 5.2.6

Then why do we have ~4.8?

**Comment by JeroenDeDauw** [comment]
Ugh, needs rebase due to https://github.com/wmde/FundraisingStore/pull/87

**Pull request merged**